styleguide/derek/pattern-components/resourceCenter-resource/_resourceCenter-resource.scss

```diff
@@ -58,10 +58,14 @@
 .resource-descWrapper {
   @include make-xs-column(12);
   margin-top: 15px;
+  padding-bottom: 25px;
 }
 
 .resource-linkWrapper {
   @include make-xs-column(12);
+  bottom: 5px;
+  padding-bottom: 10px;
+  position: absolute;
```

Comment:
If we're changing this to an absolutely positioned element, is the `@include make-xs-column(12);` stuff still needed?  Seems like to the two are at odds with each other.